bom/application/pom.xml

```diff
@@ -3051,26 +3047,6 @@
                 <version>${opentracing.version}</version>
                 <type>test-jar</type>
             </dependency>
-            <dependency>
-                <groupId>io.opentracing.contrib</groupId>
-                <artifactId>opentracing-jaxrs2</artifactId>
-                <version>${opentracing-jaxrs.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.opentracing.contrib</groupId>
-                <artifactId>opentracing-concurrent</artifactId>
-                <version>${opentracing-concurrent.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.opentracing.contrib</groupId>
-                <artifactId>opentracing-web-servlet-filter</artifactId>
-                <version>${opentracing-web-servlet-filter.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.opentracing.contrib</groupId>
-                <artifactId>opentracing-tracerresolver</artifactId>
-                <version>${opentracing-tracerresolver.version}</version>
-            </dependency>
```

Comment:
@radcortez should we start working on jdbc too? We have been unable to upgrade for a while due to native issues and the current version doesn't work well with Oracle. I have a few ideas on how to fix the native issue.

Reply:
Yes I know, this was just the first step to see if everything is ok.

We should move the remaining code to SR OpenTracing and start fixing any issues there.

Reply:
Do I understand things correctly if I say they were optional before and the users had to add them manually and that now they are always around?

Reply:
Not sure why they were part of the BOM. I guess to make sure there was no version conflict, these dependencies were only required by SR OpenTracing as far as I know.

Reply:
Yes, they were part of the BOM for the alignment.

How do you handle the new ones? I'm still unclear if they always come transitively or if the user has to add them manually?

Reply:
No, they come transitively from SR OpenTracing. All the code was copied to a single dependency: https://github.com/smallrye/smallrye-opentracing/tree/main/contrib/src/main/java/io/smallrye/opentracing/contrib

Reply:
Understood.